content/en/docs/plugin-guide/pf4j-deploy-example.md

```diff
@@ -0,0 +1,249 @@
+
+
+    # spec.spinnakerConfig.profiles - This section contains the YAML of each service's profile
+    profiles:
+      gate:    # is the contents of ~/.hal/default/profiles/gate.yml
+        spinnaker:
+          extensibility:
+            deck-proxy:
+              enabled: true
+              plugins:
+                Armory.RandomWaitPlugin:
+                  enabled: true
+                  version: 1.1.14
+            repositories:
+              examplePluginsRepo:
+                url: https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json
+```
+
+This example shows plugin configuration in the `spec.spinnakerConfig.profiles` section and the Deck proxy in the `spec.spinnakerConfig.profiles.gate` section:
+
+
+```yaml
+spec:
+  spinnakerConfig:
+    # spec.spinnakerConfig.profiles - This section contains the YAML of each service's profile
+    profiles:
+      gate:    
+        spinnaker:
+          extensibility:
+            deck-proxy: # you need this for plugins with a Deck component
+              enabled: true
+              plugins:
+                Armory.RandomWaitPlugin:
+                  enabled: true
+                  version: 1.1.14
+            repositories:
+              examplePluginsRepo:
+                url: https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json
+      orca:
+        spinnaker:
+          extensibility:
+    		   plugins:
+    		     Armory.RandomWaitPlugin:
+    			    enabled: true
+    			    version: 1.1.14
+    			    extensions:
+    				   id: armory.randomWaitStage
+    				   enabled: true
+    				   config:
+    				     defaultMaxWaitTime: 15
+    		   repositories:
+    		     examplePluginsRepo:
+    			    id: examplePluginsRepo
+    			    url: https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/plugins.json
+```
+
+Note: `repositories`, `plugins`, and `deck-proxy` are all at the same level, which is directly below `extensibility`.
+
+## Redeploy Armory
+
+From the `spinnaker-operator` directory:
+
+```bash
+kubectl -n <namespace> apply -f deploy/spinnaker/basic/SpinnakerService.yml
+```
+
+You can check pod status by executing:
+
+```bash
+kubectl -n <namespace> get pods
+```
+
+## Access the RandomWait plugin in the UI
+
+The RandomWait stage appears in the **Type** select list when you create a new Pipeline stage.
+
+{{< figure src="/images/plugins/randomWaitTypeUI.png" caption="Random Wait stage in Type select list" >}}
+
+{{< figure src="/images/plugins/randomWaitStageUI.png" caption="Random Wait stage after it has been selected and the configuration panel is visible." >}}
+
+
+## Troubleshooting
+
+If the plugin doesn't appear in the **Type** select list, check the following logs:
+
+* Orca, for the plugin backend
+
+  ```bash
+  kubectl -n spinnaker-operator logs -f <orca-pod-name>
```

Comment:
This doesn't look like the correct namespace for an orca pod. This should be the namespace where spinnaker is deployed.

Reply:
Michael - when using Operator in basic mode (the easiest for playing around since that mode doesn’t require Kustomize background), you must install Spinnaker in the spinnaker-operator namespace where the operator is installed. I emphasize that distinction with bold or italics  where I mentioned it the first time and out a reminder in the troubleshooting section.